### PR TITLE
docs: Update docker-compose file

### PR DESCRIPTION
**Pull request by rzr**
Use renamed file docker-compose-nexus-mongo.yml

Change-Id: I95dbd8d2ba8bb84d7685e113cb5ff80fde83aac1
Signed-off-by: Philippe Coval <rzr@users.sf.net>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://wiki.edgexfoundry.org/display/FA/Contributor%27s+Guide
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number:

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## Other information

### CODE REVIEW

**Comment by bnevis-i** [comment]
Merge confict.  Rebase and remove change to go.mod

**Comment by rzr** [comment]
It looks that it has been updated since:
8b5ce57112cfef5b142aa0b975f49714758365e2
 #2529

++<<<<<<< HEAD
 +https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/releases/nightly-build/compose-files/docker-compose-nexus-redis.yml
++=======
+ https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/releases/nightly-build/compose-files/docker-compose-nexus-mongo.yml
++>>>>>>> 3b1d9529... docs: Update docker-compose file

Any preference to use redis over mango ?

**Comment by mkbhanda** [comment]
Geneva onwards the default is "Redis" as opposed to "Mongo".
So preference Redis over mongo.
Thanks for all the hard work!

File: README.md [hunk 1]
```diff
@@ -13,7 +13,8 @@ They can be launched easily with **docker-compose**.
 The simplest way to get started is to fetch the latest docker-compose.yml and start the EdgeX containers:
 
 ```sh
-wget https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/releases/edinburgh/compose-files/docker-compose-edinburgh-1.0.1.yml
+wget -O docker-compose.yml \
+https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/releases/nightly-build/compose-files/docker-compose-nexus-mongo.yml
```
Comment (bnevis-i) [inline README.md:17]:
```suggestion
https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/releases/nightly-build/compose-files/docker-compose-nexus-redis.yml
```

**Comment by rzr** [comment]
So my change is no more needed I am closing , feel free to comment if needed

**Pull request closed without merging**